paddle/fluid/framework/ir/graph_pattern_detector.cc

```diff
@@ -964,6 +964,39 @@ PDNode *patterns::ElewiseAddActInplaceGrad::operator()(
   return ele_add_grad;
 }
 
+PDNode *patterns::ConvBias::operator()(
```

Comment:
As I've mentioned - setting of this parameter occurs after the passes are applied.

Reply:
Got it. Maybe we should implement MKLDNN OP Placement PASS at first to avoid this problem.

Reply:
I add #13909 to run 2 tests.

Reply:
Good job! If it gets accepted, I'll just remove non-mkldnn passes at 
https://github.com/PaddlePaddle/Paddle/pull/13909/files#diff-2e425e802cdeb2a590bc1e5ac9e34f11R23 
and 
https://github.com/PaddlePaddle/Paddle/pull/13909/files#diff-2e425e802cdeb2a590bc1e5ac9e34f11R23 
if the `_use_mkldnn` parameter will be set to true.

Reply:
How can I detect in graph detector whether the MKL-DNN passes are to be used or not when by default all `use_mkldnn` attributes are set to false?

Reply:
@Superjomn @tensor-tang Do you have some suggestions about it?

Reply:
@wojtuss has created a pass that sets mkldnn attribute to operators. He will create a pull request today. If it is accepted, @Sand3r- you could use it.

Reply:
@luotao1 Since this is the blocker for the #13485 could we possibly follow this scenario?:
1. Once your #13909 will get merged, I'll apply the temporarily fix for the solution by disabling the MKL-DNN Passes when the `_use_mkldnn` flag is off, by adding them to exception list in config.
2. When @wojtuss will prepare his PR with pass setting use_mkldnn flag to true, I'll remove these pass exceptions from tests and adjust the code accordingly to your will - that is - the value of use_mkldnn parameter will be checked at the stage of detecting patterns in `graph_pattern_detector.cc`.

Reply:
@Sand3r- Since @wojtuss will create a pull request today, could you wait a few more hours?

Reply:
As long as that solution is satisfactory for you, I see no trouble :wink: